docs/validate/testnet.mdx

```diff
@@ -110,22 +110,7 @@ For more information on seeds and peers, you can the Tendermint
 ### Add Persistent Peers
 
 We can set the [`persistent_peers`](https://docs.tendermint.com/v0.34/tendermint-core/using-tendermint.html#persistent-peer) field in `~/.evmosd/config/config.toml` to specify peers that your node will maintain persistent
- connections with. You can retrieve them from the list of
-available peers on the [`testnets`](https://github.com/evmos/testnets) repo.
-
-A list of available persistent peers is also available in the `#find-peers` channel in the
-[Evmos Discord](https://discord.gg/evmos). You can get a random 10 entries from the `peers.txt`
- file in the `PEERS` variable by running the following command:
-
-```bash
-PEERS=`curl -sL https://raw.githubusercontent.com/evmos/testnets/main/evmos_9000-4/peers.txt | sort -R | head -n 10 | awk '{print $1}' | paste -s -d, -`
-```
```

Comment:
The discord channel no longer exists, is OK to remove it. But the txt file is still there. Please, only remove the discord section